pyproject.toml

```diff
@@ -68,6 +68,7 @@ sqlalchemy = "^2.0.0"
 sqlalchemy_utils = "*"
 sqlmodel = "0.0.18"
 importlib_metadata = { version = "<=7.0.0", python = "<3.10" }
+ipynbname = "2024.1.0.0"
```

Comment:
@schustmi maybe we could just take what we need from here directly as a util funtion and acknowledge? this file is the entirety of the package...

https://github.com/msm1089/ipynbname/blob/master/ipynbname/__init__.py